target/linux/layerscape/image/armv8_64b.mk

```diff
@@ -102,7 +102,8 @@ define Device/fsl_ls1043a-rdb
   DEVICE_PACKAGES += \
     layerscape-fman \
     tfa-ls1043a-rdb \
-    fmc fmc-eth-config
+    fmc fmc-eth-config \
+    kmod-hwmon-ina2xx kmod-hwmon-lm90
```

Comment:
This Makefile seems to have a one-package-per-line scheme, please maintain that for your additions.

Reply:
It is mixed. Some packages with the same first letter are together:  ```fmc fmc-eth-config``` or  ```kmod-gpio-pca953x kmod-input-gpio-keys-polled \```. Some not.
But I will change it, if you want.

Reply:
I see what you mean. However, it appears to me that the original idea was to keep one line per packages, and I'd stick to that for adding new stuff (actually, that scheme is more convenient for some use-cases).

The "fmc fmc-eth-config" is at least consistently deviating from that, and I just would ignore it. The "kmod-gpio-pca953x kmod-input-gpio-keys-polled" actually should be converted into two lines, but I don't think it would be worth a commit just for that.